File: Csharp/BankOcr-Kata/BankOcr.cs
using System;
using System.Collections.Generic;

namespace Kata
{
    /// <summary>
    /// Example for the outside interface of the API we need to create.
    /// </summary>
    public class BankOcr
    {

        public IList<AccountNumber> Parse(IEnumerable<string> rawLines)
        {
            throw new NotImplementedException("not implemented");
        }

    }
}

File: Python/kata/bankocr.py
# coding=utf-8
from kata.accountnumber import AccountNumber


class BankOcr(object):
    """Example for the outside interface of the API we need to create."""

    def __init__(self):
        pass

    def parse(self, raw_lines):
        # TODO return an array of AccountNumber
        raise NotImplementedError("not implemented")
